docs/content/en/_index.md

```diff
@@ -18,19 +18,17 @@ Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandr
 
 ## Features for single- and multi-cluster Kubernetes environments
 
-| K8ssandra v1.4.x      | K8ssandra Operator v1.0.0 |
+| K8ssandra       | K8ssandra Operator for enhanced capabilities |
 | ----------- | ----------- |
-| K8ssandra v1.4.x is our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
-| Start in the K8ssandra v1.4.x [install](https://docs-staging-v1.k8ssandra.io/install/local/) topics, which include the steps for single-cluster installs on local or cloud-provider Kubernetes platforms. | Start in the K8ssandra Operator v1.0.0 [local install]({{< relref "install/local" >}}) topic, which focuses on multi-cluster or single-cluster installs on local dev  **kind** Kubernetes, using helper scripts, `helm`, or `kustomize`.
+| K8ssandra v1.x was our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
+| Start in the K8ssandra v1.4.x [install](https://docs-staging-v1.k8ssandra.io/install/local/) topics, which include the steps for single-cluster installs on local or cloud-provider Kubernetes platforms. | For enhanced capabilities, start in the K8ssandra Operator v1.0.0 [local install]({{< relref "install/local" >}}) topic, which focuses on single- or multi-cluster deployments on local dev  **kind** Kubernetes clusters, using the provided `make` commands, `helm`, or `kustomize`.
```

Comment:
This is a question that I grappled with writing the release blog... should we be leading with K8ssandra Operator as THE way from here on out, trying to actively push people to using it over the original charts.

Obviously that's up to each users opinion in terms of stability and functionality, but I'm leaning towards, let's start leading with the operator as the way (insert Mandelorian emoji here 😉 ).

With that in mind should we perhaps reword this a bit to introduce K8ssandra Operator first and also just present it as the default - and not just the "for advanced cases" sort of solution?

Reply:
Yeah let me go with that (ha, Mandelorian), K8ssandra Operator as the recommended Kubernetes + Cassandra solution.